__tests__/app.test.js

```diff
@@ -12,19 +12,42 @@ afterAll(() => {
   return connection.end();
 });
 
-
 describe("GET/api/topics", () => {
-    test("200: responds with an array of topic objects with their slug and description", () => {
-        return request(app)
-        .get("/api/topics")
-        .expect(200)
-        .then((res) => {
-            const { topics } = res.body
-           expect(topics).toHaveLength(3)
-           topics.forEach((topic) => {
-            expect(topic).toHaveProperty("slug", expect.any(String))
-            expect(topic).toHaveProperty("description", expect.any(String) )
-           })
-        })
-    })
-})
+  test("200: responds with an array of topic objects with their slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const { topics } = res.body;
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug", expect.any(String));
+          expect(topic).toHaveProperty("description", expect.any(String));
+        });
+      });
+  });
+});
+
+describe("GET/api", () => {
+  test("200: responds with all available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        const { endpoints } = res.body;
+        expect(endpoints).toHaveProperty("GET /api");
```

Comment:
This is alright for now, but when we start adding more endpoints you might need to change the test which is not ideal. Alternatively we can require the enpoints json file (like: const endpointsJson = require("../.......something.json") and we can expect(endpoints).toEqual(endpointsJson) 🙂